<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenuDialog</class>
 <widget class="QDialog" name="StartMenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trivial Purfuit Menu</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>91</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="new_game_button">
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>StartMenuDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
